convert2rhel/utils.py

```diff
@@ -568,3 +568,47 @@ def is_dir_empty(path):
     for path in rh_release_paths:
         if os.path.exists(path) and is_dir_empty(path):
             os.rmdir(path)
+
+
+def hide_secrets(args):
+    """
+    Replace secret values with asterisks.
+
+    This function takes a list of arguments which will be passed to
+    subscription-manager on the command line and returns a new list
+    that has any secret values obscured with asterisks.
+
+    :arg args: An argument list for subscription-manager which may contain
+        secret values.
+    :returns: A new list of arguments with secret values hidden.
+    """
+    obfuscation_string = "*" * 5
+    secret_args = frozenset(("--password", "--activationkey", "--token"))
```

Comment:
```suggestion
    secret_args = frozenset(("--password", "--activationkey", "--token", "-p", "-k"))
```
The shortened options do not obfuscate the values @SpyTec

Reply:
This was apparently fixed in #612 and I didn't realise. Fixed now though